src/libhirte/common/cfg.c

```diff
@@ -133,6 +138,96 @@ int cfg_load_from_file(struct config *config, const char *config_file) {
         return ini_parse(config_file, parsing_handler, config);
 }
 
+void add_file_to_list_and_sort(char *file_location) {
```

Comment:
Wouldn't it be better to use list macros from https://github.com/containers/hirte/blob/main/src/libhirte/common/list.h ?